orderpage.html

```diff
@@ -42,10 +38,7 @@ <h3>Select up to two jams:</h3>
                 <input type="checkbox" name="jams" value="Marionberry"/> Marionberry
                 <input type="checkbox" name="jams" value="Orange Marmalade"/> Orange Marmalade
                 <input type="checkbox" name="jams" value="Mixed Berry"/> Mixed Berry
-          </legend>
-        </fieldset>
-        <fieldset>
-          <legend>
+
             <h3>Want Extras?</h3>
               <br/>
```

Comment:
Just a reminder, you don't need break tags unless you're in the middle of a paragraph or just after an inline tag. All h1-6 tags, inputs, p, etc. are block tags so they start a new a new line anyways :-)